course/templates/course/course_instances.html

```diff
@@ -0,0 +1,29 @@
+{% extends "base.html" %}
+{% load i18n %}
+{% load course %}
+
+{% block title %}{% trans "Course instances" %} | {{ block.super }}{% endblock %}
+{% block view_tag %}instances{% endblock %}
+
+{% block content %}
+  <div class="frontpage-panel">
```

Comment:
Does this HTML template use spaces for indentation? I think the new style was to use tabs, though it is not a big issue. It would be nice to use the correct, consistent coding style in new files.

Reply:
Ups, I forgot to activate the editorconfig extension in VS Code